test/fixtures/output/configuration/list-windows.golden

```diff
@@ -0,0 +1,20 @@
+          Name          |                       Value                        | Read-Only
+------------------------+----------------------------------------------------+------------
+  context_states        | map[string]*config.ContextState                    | true
+                        | with keys: [context1]                              |
+  contexts              | map[string]*config.Context                         | true
+                        | with keys: [context1]                              |
+  credentials           | map[string]*config.Credential with keys:           | true
+                        | [username-cli-mock-email@confluent.io-http://test] |
+  current_context       | context1                                           | false
+  disable_feature_flags | false                                              | false
+  disable_plugins       | false                                              | false
+  disable_plugins_once  | false                                              | false
+  disable_update_check  | false                                              | false
+  disable_updates       | false                                              | false
+  local_ports           |                                                    | true
```

Comment:
Looks like this is empty?

Reply:
The mock `config` I'm using for tests doesn't have `local` setup. I have to look at `confluent local` to see if there's a mock or just make one myself.

Reply:
(It should say `[]` if it doesn't exist)